shared/people/wot-task/index.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react'
+import * as Container from '../../util/container'
+import * as Kb from '../../common-adapters'
+import * as PeopleGen from '../../actions/people-gen'
+import * as RPCTypes from '../../constants/types/rpc-gen'
+import * as Styles from '../../styles'
+import PeopleItem, {TaskButton} from '../item'
+import {WotStatusType} from '../../constants/types/rpc-gen'
+
+type Props = {
+  key: string
+  onClickUser: (username: string) => void
+  status: RPCTypes.WotStatusType
+  vouchee: string
+  voucher: string
+}
+
+const makeButtons = (
+  props: Props,
+  onDismiss: (voucher: string, vouchee: string) => void
+): Array<TaskButton> => {
+  const dismissButton = {
+    label: 'Dismiss',
+    mode: 'Secondary',
+    onClick: () => onDismiss(props.voucher, props.vouchee),
+  } as TaskButton
+  switch (props.status) {
+    case WotStatusType.proposed:
+      return [
+        {
+          label: 'Review claim',
+          onClick: () => props.onClickUser(props.vouchee),
+        },
+        dismissButton,
+      ]
+    case WotStatusType.accepted:
+      return [
+        {
+          label: 'Review claim',
+          onClick: () => props.onClickUser(props.vouchee),
+        },
+        dismissButton,
+      ]
+    case WotStatusType.rejected:
+      return [
+        {
+          label: 'Edit claim',
+          onClick: () => props.onClickUser(props.vouchee),
+        },
+        dismissButton,
+      ]
+    default:
+      return []
+  }
+}
+
+const makeMessage = (props: Props) => {
+  const connectedUsernamesProps = {
+    colorFollowing: true,
+    inline: true,
+    joinerStyle: {
+      fontWeight: 'normal',
+    },
+    onUsernameClicked: 'profile',
+    type: 'BodyBold',
+    underline: true,
+  } as const
+  const voucherComponent = (
+    <Kb.ConnectedUsernames
+      {...connectedUsernamesProps}
+      usernames={props.voucher}
+      onUsernameClicked={props.onClickUser}
+    />
+  )
+  const voucheeComponent = (
+    <Kb.ConnectedUsernames
+      {...connectedUsernamesProps}
+      usernames={props.vouchee}
+      onUsernameClicked={props.onClickUser}
+    />
+  )
+  switch (props.status) {
+    case WotStatusType.proposed:
+      return <Kb.Text type="Body">{voucherComponent} submitted an entry to your web of trust.</Kb.Text>
+    case WotStatusType.accepted:
+      return <Kb.Text type="Body">{voucheeComponent} accepted your entry into their web of trust.</Kb.Text>
+    case WotStatusType.rejected:
+      return <Kb.Text type="Body">{voucheeComponent} rejected your entry into their web of trust.</Kb.Text>
+    default:
+      return <Kb.Text type="Body">unknown.</Kb.Text>
+  }
+}
+
+const WotTask = React.memo((props: Props) => {
```

Comment:
Did this always have `React.memo` on it?

Since one of the props is a function `onClickUser`, I suspect that the shallow equal comparison done by `memo` will return `false` on each render since a new function ID will be created.

Can either remove `memo` since it might not help in the way you want, or just reimplement `onClickUser` in `WotTask` and `dispatch` the same action.